bin/bs-bench.scala

```diff
@@ -117,24 +121,33 @@ object Main
 
             if (updatePlots) command += s" --update-plots"
 
-            if (repetitions.isDefined) command += s" --repetitions ${repetitions.get}"
-
             val start = System.nanoTime
 
             sys.process.Process(command, new java.io.File(path)).!!
```

Comment:
I'm not sure exactly where, but you should redirect standard error so it doesn't go to console
see e.g. https://alvinalexander.com/scala/how-to-redirect-stdout-stdin-external-commands-in-scala/

Here it is 
`bin$ ./bs-bench.scala  --old /Applications/NetLogo\ 6.3.0 --new /Applications/NetLogo\ 6.4.0-beta1 --model /Users/aab1638/ModelsNetLogo/anisogamy/seam.nlogo --experiment mutation-low-500-1 --threads 8 --vary-plots --output anisogamy-500-1.csv`
```

> Testing NetLogo 6.3.0...
> unknown argument: --update-plots
> Testing NetLogo 6.3.0...
> Trial 1 of 1 completed in 0.1081836 minutes.
> Testing NetLogo 6.4.0-beta1...
> Trial 1 of 1 completed in 0.12211842 minutes.
> Testing NetLogo 6.4.0-beta1...
> Trial 1 of 1 completed in 0.10189773 minutes.

```